euth/ideas/models.py

```diff
@@ -20,6 +22,11 @@ def get_absolute_url(self):
         from django.core.urlresolvers import reverse
         return reverse('idea-detail', args=[str(self.slug)])
 
+    def clean(self):
+        super().clean()
+        self.description = html_transforms.clean_html_field(
+            self.description)
```

Comment:
Of course it is right to do the transformation of the description in `Model.clean()`. But you have to remember that `Model.clean()` is not called if you just call `Model.save()` (see [Validating objects](https://docs.djangoproject.com/en/1.8/ref/models/instances/#validating-objects)). Only a `ModelForm` will call `Model.clean()`. So if you want to make sure that the description is always transformed you should move the code to `Model.save()`.